tests/snapshot_tests/snapshot_apps/scroll_to_center.py

```diff
@@ -5,32 +5,33 @@
 
 class MyApp(App[None]):
     CSS = """
-    VerticalScroll, HorizontalScroll {
+    VerticalScroll {
         border: round $primary;
     }
     #vertical {
         height: 21;
     }
     HorizontalScroll {
+        border: round $secondary;
         height: auto;
     }
+    Label {
+        height: auto;
+        width: auto;
+    }
     """
 
     def compose(self) -> ComposeResult:
-        with VerticalScroll():
-            yield Label(("SPAM\n" * 205)[:-1])
-            with VerticalScroll():
-                yield Label(("SPAM\n" * 53)[:-1])
-                with VerticalScroll(id="vertical"):
-                    yield Label(("SPAM\n" * 78)[:-1])
-                    with HorizontalScroll():
-                        yield Label(("v\n" * 17)[:-1])
-                        yield Label("@" * 302)
-                        yield Label("[red]>>bullseye<<[/red]", id="bullseye")
-                        yield Label("@" * 99)
-                    yield Label(("SPAM\n" * 49)[:-1])
-                yield Label(("SPAM\n" * 51)[:-1])
-            yield Label(("SPAM\n" * 59)[:-1])
+        yield Label(("SPAM\n" * 53)[:-1])
```

Comment:
Simplified this a little because I was struggling to understand it vs what was on screen. It still has multiple levels of nested scrolling containers, both vertical and horizontal.